nix/release.nix

```diff
@@ -46,7 +46,7 @@ let
     hschain-logger         = callInternal hsPkgs "hschain" ../hschain-logger         {} "";
     hschain-net            = callInternal hsPkgs "hschain" ../hschain-net            {} "";
     hschain-config         = callInternal hsPkgs "hschain" ../hschain-config         {} "";
-    hschain-pow-func       = callInternal hsPkgs "hschain" ../proof-of-work          {} "";
+    hschain-pow-func       = callInternal hsPkgs "hschain" ../proof-of-work/full-mining {} "";
```

Comment:
I think `../proof-of-work/lightweight-check` is missing here (unless it's not part of build yet)